src/__test__/index.test.ts

```diff
@@ -1,4 +1,4 @@
-import {uploadNode} from './index'
+import {uploadNode} from '../index'
```

Comment:
out of interest, why have you decided to move away from co-located tests?

Reply:
it just seemed a bit messy imo. they are still somewhat colocated – all tests for files in a particular directory will be in a the sibling directory `__test__` rather than some global location